### DISCUSSION TITLE
Wrong overspent value with 2 budgets based on different periods?

### DISCUSSION THREAD

**Discussion by Thulkash** [firefly-iii/firefly-iii#7748] [Firefly III]
Hello,

Today, I created:
- 2 budgets: 1 for annual bills (2500€), and one for monthly bills (1040,46€).
- 2 expenses: 1 in annual budget (875€) and 1 in monthly budget (841,66€).

The overspent value, in the dashboard pane, seems to be not correct: -463,87€ (1040,46 - 875 - 841,66)
The transaction of 875€ should not be included in the overspent value, because this transaction is not related to the monthly budget.

![Capture d’écran 2023-07-11 223546](https://github.com/firefly-iii/firefly-iii/assets/46101710/dc2fc595-61f7-477c-9d04-fd48158f7e2c)

![Capture d’écran 2023-07-11 223602](https://github.com/firefly-iii/firefly-iii/assets/46101710/1c67e7df-7c69-46b5-bb70-bf0983f2c7e5)

**Comment by JC5** [comment]
>  The transaction of 875€ should not be included in the overspent value, because this transaction is not related to the monthly budget.

If it is spent in the current time period it will be included.

**Reply by Thulkash** [reply]
So, annual budgets are useless?
